luabind::class_<::addr_info>("addr_info")
.def(luabind::constructor<>())
.def("CopyFrom", (void(::addr_info::*)(const ::addr_info &))&::addr_info::CopyFrom)
.def("ParseFromString", &::addr_info::ParseFromString)
.def("clear_addr", &::addr_info::clear_addr)
.def("addr", &::addr_info::addr)
.def("set_addr", static_cast<void(::addr_info::*)(const char *)>(&::addr_info::set_addr))
.def("clear_port", &::addr_info::clear_port)
.def("port", &::addr_info::port)
.def("set_port", &::addr_info::set_port)
,luabind::class_<::addr_map>("addr_map")
.def(luabind::constructor<>())
.def("CopyFrom", (void(::addr_map::*)(const ::addr_map &))&::addr_map::CopyFrom)
.def("ParseFromString", &::addr_map::ParseFromString)
.def("clear_locals", &::addr_map::clear_locals)
.def("add_locals", &::addr_map::add_locals)
.def("mutable_locals", (::addr_info*(::addr_map::*)(int))(&::addr_map::mutable_locals))
.def("locals_size", &::addr_map::locals_size)
.def("clear_remotes", &::addr_map::clear_remotes)
.def("add_remotes", &::addr_map::add_remotes)
.def("mutable_remotes", (::addr_info*(::addr_map::*)(int))(&::addr_map::mutable_remotes))
.def("remotes_size", &::addr_map::remotes_size)
,luabind::class_<::configure>("configure")
.def(luabind::constructor<>())
.def("CopyFrom", (void(::configure::*)(const ::configure &))&::configure::CopyFrom)
.def("ParseFromString", &::configure::ParseFromString)
.def("clear_maps", &::configure::clear_maps)
.def("add_maps", &::configure::add_maps)
.def("mutable_maps", (::addr_map*(::configure::*)(int))(&::configure::mutable_maps))
.def("maps_size", &::configure::maps_size)
.def("clear_run_as_daemon", &::configure::clear_run_as_daemon)
.def("run_as_daemon", &::configure::run_as_daemon)
.def("set_run_as_daemon", &::configure::set_run_as_daemon)
.def("clear_nb_services", &::configure::clear_nb_services)
.def("nb_services", &::configure::nb_services)
.def("set_nb_services", &::configure::set_nb_services)
.def("clear_server_crash_run", &::configure::clear_server_crash_run)
.def("has_server_crash_run", &::configure::has_server_crash_run)
.def("server_crash_run", &::configure::server_crash_run)
.def("set_server_crash_run", static_cast<void(::configure::*)(const char *)>(&::configure::set_server_crash_run))
